src/components/Timer/timer.js

```diff
@@ -65,6 +66,17 @@ export default function Timer(props) {
     }
   }
 
+  function resetPomodoroCount() {
+    // Display an alert to confirm the reset
+    const isConfirmed = window.confirm("Are you sure you want to reset the pomodoro count?");
+
+    // If user confirms, reset the count
+    if (isConfirmed) {
+      setCycles(0);
```

Comment:
You should also update the local storage here.

Reply:
Fixed! 👍🏼